tools/simde_update.py

```diff
@@ -67,10 +67,11 @@ def main():
       f.write(re.sub(SIMDE_FILE_HEADER_RE, r'\1\2\3', neon_h_buf, count=0, flags=re.MULTILINE))
       f.write("#undef SIMDE_ARM_NEON_A32V7_ENABLE_NATIVE_ALIASES\n")
       f.write("#undef SIMDE_ARM_NEON_A64V8_ENABLE_NATIVE_ALIASES\n")
-      print("'system/include/compat/arm_neon.h' updated")
     except Exception:
       print("error writing 'system/include/compat/arm_neon.h'")
       return 1
+  os.system("sed -i '/HEDLEY_EMSCRIPTEN_VERSION HEDLEY_VERSION_ENCODE(__EMSCRIPTEN_major__, __EMSCRIPTEN_minor__, __EMSCRIPTEN_tiny__)/i #include <emscripten/version.h>' system/include/compat/arm_neon.h")
```

Comment:
Can you do this part in python instead of forking out to sed?

Reply:
Done!